core/config.py

```diff
@@ -1,13 +1,18 @@
 import sys
+from os import getenv
+from itertools import chain
 from copy import deepcopy
 from pathlib import Path
 
 import yaml
 
 # from deeppavlov import configs
 
+TELEGRAM_TOKEN = ''
+TELEGRAM_PROXY = ''
+
 DB_NAME = 'test'
-HOST = 'mongo'
+HOST = '127.0.0.1'
```

Comment:
As soon as the host changed, including `mongo` container in docker-compose.yml is broken (see the logic in `generate_composefile.py`, line 99).